docs/media_features.md

```diff
@@ -81,4 +81,6 @@
 |SKL/BXT/KBL/CFL/WHL/CML| Input  |  Y   |  Y   |  Y   |  Y   |  Y   |      |      |      |      |
 |                       | Output |  Y   |  Y   |  Y   |      |  Y   |      |      |      |      |
 |      ICL              | Input  |  Y   |  Y   |  Y   |  Y   |  Y   |  Y   |  Y   |  Y   |  Y   |
+|                       | Output |  Y   |  Y   |  Y   |  Y   |  Y   |      |  Y   |  Y   |  Y   |
+|      TGL              | Input  |  Y   |  Y   |  Y   |  Y   |  Y   |  Y   |  Y   |  Y   |  Y   |
```

Comment:
Where JSL and EHL belong to?

Reply:
@FurongZhang please help to check the vpp feature list in the table especially of EHL/JSL